eng/references.targets

```diff
@@ -23,21 +23,12 @@
       <SetConfiguration Condition="'$(RuntimeFlavor)' == 'Mono' and
                                    '$(Configuration)' != '$(MonoConfiguration)'">Configuration=$(MonoConfiguration)</SetConfiguration>
     </ProjectReference>
-    <!-- If a CoreLib ProjectReference is present, make all P2P assets non transitive. -->
+    <!-- If a CoreLib src/ref ProjectReference is present, make all P2P assets non transitive. -->
```

Comment:
nit: I don't think we should differentiate between the kind of ProjectReference, especially as we never directly P2P the ref CoreLib.

```suggestion
    <!-- If a CoreLib ProjectReference is present, make all P2P assets non transitive. -->
```

Reply:
yup, that was a leftover from an earlier iteration